proto/proto/account.proto

```diff
@@ -19,7 +19,7 @@ message AccountSummary {
     AccountId account_id = 1;
 
     // The current account commitment or zero if the account does not exist.
-    digest.Digest account_commitment = 2;
+    word.Word account_commitment = 2;
```

Comment:
Flagging this too. Just not sure the proto needs to change. Unsure who to @ to ask.

Reply:
I would probably keep it as `Digest` - at least for now. I think almost all uses of this in the protobuf files are actual digests (i.e., result of hashing something).